DB/common.php

```diff
@@ -1873,7 +1873,7 @@ function dropSequence($seq_name)
      *
      * @see PEAR_Error
      */
-    function &raiseError($code = DB_ERROR, $mode = null, $options = null,
+    static function raiseError($code = DB_ERROR, $mode = null, $options = null,
```

Comment:
This smells a bit problematic to me
 * The parent class's `PEAR::raiseError()` is sorta polymorphic (static *or* nonstatic). In PEAR 1.10, this is done with magic functions, which is probably why it's not producing a syntax error.
 * There are only a couple classes which would inherit this function (`DB_mysql`, `DB_mysqli`, etc), and I'm not sure we haven't test-coverage that purposefully triggers a SQL error.

Maybe the right thing to do is override `protected static function _raiseError()`? (In PEAR 1.10, this is the real implementation shared by the static and non-static variants of `raiseError`.)